helpme/views/helpme.py

```diff
@@ -77,18 +54,21 @@ def get_paginate_by(self, queryset):
         return self.paginate_by
 
     def get_queryset(self, **kwargs):
+        site = Site.objects.get_current()
         # admin
-        if self.request.user.has_perm('helpme.see_all_tickets'):
+        if self.request.user.is_staff or self.request.user.is_superuser:
             queryset = Ticket.objects.all()
+        elif self.request.user.has_perm('helpme.see_all_tickets'):
+            queryset = Ticket.objects.filter(site=site)
```

Comment:
Not a big deal but for consistency we should probably add an "on_site" model manager.  The manager is part of Django so it should be easy to drop in.

```
from django.contrib.sites.managers import CurrentSiteManager

...

objects = models.Manager() 
on_site = CurrentSiteManager()
```